bundles/org.openhab.binding.dominoswiss/README.md

```diff
@@ -0,0 +1,110 @@
+# Dominoswiss Binding
+
+This binding allows the controll of rollershutters, using an egate as gateway and dominoswiss radio receivers. The egate-gateway is connectet via ethernet to openhab and sends it's commands via radio to all rollershutters. See https://www.brelag.com/ for more information.
+
+
+## Supported Things
+
+EGate: Dominoswiss EGate Server. The egate is the gateway which sends the commands to the connected rollershutters. 
+Blind: represents one rollershutter, that can be controlled via EGate
+
+## Discovery
+
+Unfortunately no automatic discovery is possible due to protocol restrictions. Every rollershutter must be known by egate and can be called by it's number of storage-place on egate gateway.
+
+
+## Binding Configuration
+
+_If your binding requires or supports general configuration settings, please create a folder ```cfg``` and place the configuration file ```<bindingId>.cfg``` inside it. In this section, you should link to this file and provide some information about the options. The file could e.g. look like:_
+
+```
+# Configuration for the Philips Hue Binding
+#
+# Default secret key for the pairing of the Philips Hue Bridge.
+# It has to be between 10-40 (alphanumeric) characters
+# This may be changed by the user for security reasons.
+secret=openHABSecret
+```
+
+_Note that it is planned to generate some part of this based on the information that is available within ```src/main/resources/OH-INF/binding``` of your binding._
+
+_If your binding does not offer any generic configurations, you can remove this section completely._
```

Comment:
I recommend you remove it completely as this looks like it was auto generated a very long time ago by an old skeleton script ?!?!?